articles/prototyping-machine-learning-models-with-streamlit/index.md

```diff
@@ -0,0 +1,212 @@
+title: Prototyping Machine learning models with streamlit
+ 
+description: Most times machine learning engineers build models and will need to build a web or mobile app to prototype their models which is another stress on its own, this tutorial will show readers how to quickly prototype their ML models on web applications easily.
+ 
+### Prototyping Machine learning models with Streamlit
+ 
+![Simple](https://github.com/jamessandy/engineering-education/blob/new-article/articles/prototyping-machine-learning-models-with-streamlit/hero.jpg)
+ 
+As Machine learning Engineers or Data scientist, you are used to building cool machine learning models but probably not familiar with building mobile or web apps, or the thought of building an application to serve your model is stressful because of the whole process of building either a web or mobile app so you just leave your model(s) on your notebook without using the weights. Streamlit is a platform that can help you prototype your model(s) in very few lines of code with less stress.
+ 
+#### Tabel of Content
+1. Introduction to streamlit
+2. Streamlit setup
+3. Building sentient analyzer
+4. Conclusion
+ 
+#### Introduction to Streamlit
+ 
+Streamlit is an Open source python framework that enables developers to demo or prototype their ML models and a lot more on a web application, you can simply turn your data science scripts into a website with few lines of code. It saves time, energy, and is also simple to learn, use, and fast too. Now we’ll dive straight into how to install and setup streamlit and then build a text summarizer web app.
+ 
+##### Streamlit Set up
+ 
+The installation is very simple, just run this on your command prompt or terminal
+ 
+```python
+pip install streamlit
+streamlit hello
+```
+ 
+To import it, use the simple
+ 
+```python
+import pandas as pd
+import streamlit as st
+import plotly.express as px
+```
+ 
+And to run your streamlit app use
+ 
+```python
+streamlit run app.py
+```
+ 
+#### Building Sentiment Analyzer
+ 
+Now for this beginner’s tutorial, we will be building a sentiment analyzer model and prototype it with streamlit. The first step will be getting a Dataset, and the dataset can be found [HERE](https://drive.google.com/drive/folders/1wk0hkrRhxe6t5g390g5V5O-CUcRB6BZC?usp=sharing), next let’s create a directory and create a virtual environment, open you terminal or command prompt and type in the code below.
```

Comment:
@jamessandy The intention behind asking you to break down complex sentences into simpler ones is to ensure better readability. Consider the following sentence. Instead of the commas, you could replace them with periods.

In this beginner’s tutorial, we will build a sentiment analyzer model and prototype it with streamlit. The first step will be getting a Dataset, which is available HERE. Further, let’s create a directory and a virtual environment. Open your terminal or command prompt and type in the code below.

Periods rather than commas separate sentences. If you can change this, then this article will be ready for the final round of review. Also, instead of using, _we will be building_, use **we build**. It's more concise.